ext/mbstring/mbstring.stub.php

```diff
@@ -65,29 +65,29 @@ function mb_list_encodings(): array {}
 
 function mb_encoding_aliases(string $encoding): array {}
 
-function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer = null, string $linefeed = "\r\n", int $indent = 0): string {}
+function mb_encode_mimeheader(string $string, ?string $charset = null, ?string $transfer_encoding = null, string $linefeed = "\r\n", int $indent = 0): string {}
```

Comment:
Synonyms for "line feed" according to Wikipedia: new line, line ending, end of line, or line break. I think a few of them would qualify as a better (more well-known) name. E.g. `$newline`? If you don't agree, can you use `$line_feed`? As far as I saw it should be spelled as two words.

Reply:
I went with `$newline` here, agree that line feed is uncommon.